shelley/chain-and-ledger/formal-spec/update.tex

```diff
@@ -10,9 +10,9 @@ \section{Update Proposal Mechanism}
 software update is an acceptable future option (again, only genesis keys vote on this).
 For rules regarding the \textit{adoption} of protocol
 parameter updates, see \ref{sec:pparam-update}. For rules regarding
-adoption of new software versions see \ref{sec:software-updates}.
+adoption of new software versions see \ref{sec:software-updates}. \newline
 
-\underline{Distinction between protocol parameter and application updates.}
+\noindent \underline{Distinction between protocol parameter and application updates.}
 \begin{itemize}
 \item \textbf{Protocol parameters} (excluding $\ProtVer$, explained below): constants
```

Comment:
why distinguish the protocol version? it's just a protocol parameter like the others.